manim/utils/family.py

```diff
@@ -0,0 +1,32 @@
+import itertools as it
+
+from ..mobject.mobject import Mobject
+from ..utils.iterables import remove_list_redundancies
+
+
```

Comment:
Great idea. The next step is to document what on earth a mobject's "family" even is. It took me forever to understand this particular concept in the codebase.

Reply:
Yeah, I just removed it

Reply:
Done.